.github/workflows/community-label.yml

```diff
@@ -22,6 +22,9 @@ jobs:
             issues: write
         steps:
             - uses: actions/checkout@v3
+            # We want to delay the labeling of issues until after the awaiting triage script runs.
+            - name: Delay Labeling
+              run: sleep 4m
```

Comment:
Should we move this down below the actual check and add a condition check to this? (`if: github.event.pull_request && steps.check.outputs.is-community == 'yes'`). We don't need the workflow hanging around waiting for 4 minutes if at the end of the day we won't be adding the label anyways.

Reply:
Edit: looking at how the labeling works, it looks like we'd need to actually remove the labeling from the script (`.github/workflows/scripts/is-community-contributor.js`) if we wanted to do that.

I'll leave it up to you @rrennick  whether or not it's worth the effort. Changing my review to acceptance.